cluster/charts/crossplane/README.md

```diff
@@ -144,7 +144,7 @@ metadata:
   name: provider-gcp
   namespace: gcp
 spec:
-  package: "crossplane/provider-gcp:master"
+  package: "crossplane/provider-gcp:v0.7.0"
```

Comment:
Should these be in here?

Reply:
Do you mean in the Helm `README` specifically? This is copied here as part of `make generate` and I think we do want it there. If not I will have to adjust our `Makefile` or we will fail on `check-diff`.